src/components/stocks/Stocks.vue

```diff
@@ -11,12 +11,12 @@ export default {
   name: "Stock",
   data() {
     return {
-      stocks: [
-        { id: 1, name: "BMW", price: 110 },
-        { id: 1, name: "Google", price: 200 },
-        { id: 1, name: "Apple", price: 1250 },
-        { id: 1, name: "Twitter", price: 8 }
-      ]

```

Comment:
You have stocks defined both on computed and data. Thats should not happen, so i removed it from data